conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalytics.java

```diff
@@ -155,9 +158,11 @@ public void initializeSession() throws ConvivaAnalyticsException {
             return;
         }
 
-        if (bitmovinPlayer.getConfig().getSourceItem() == null && this.contentMetadataBuilder.getAssetName() == null) {
+        if (bitmovinPlayer.getConfig().getSourceItem() == null
```

Comment:
This will not work. I think you intended to check for `||`?

EDIT: You may also consider using `TextUtils.isEmpty(text)` for your checks.

Reply:
Thanks 🤦‍♂️